File: LittleProjects/Java/SpringHibernateCRUD/src/main/java/core/configuration/AppInitializer.java
package core.configuration;

import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer;

public class AppInitializer extends AbstractAnnotationConfigDispatcherServletInitializer {
    protected Class<?>[] getRootConfigClasses() {
        return new Class[] {App.class};
    }

    protected Class<?>[] getServletConfigClasses() {
        return new Class[] {App.class};
    }

    protected String[] getServletMappings() {
        return new String[] {"/"};
    }
}

File: LittleProjects/Scala/ScalaSpringSimpleCRUD/src/main/java/com/github/gr1f0n6x/configuration/AppInitializer.scala
package com.github.gr1f0n6x.configuration

import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer

class AppInitializer extends AbstractAnnotationConfigDispatcherServletInitializer {
  override def getServletConfigClasses = Array(classOf[App], classOf[Source])

  override def getRootConfigClasses = Array(classOf[App], classOf[Source])

  override def getServletMappings = Array("/")
}
